docs/source/autogen/lineapy.plugins.rst

```diff
@@ -16,6 +46,12 @@ lineapy.plugins.task module
 .. automodule:: lineapy.plugins.task
    :members:
 
+lineapy.plugins.taskgen module
+------------------------------
+
+.. automodule:: lineapy.plugins.taskgen
+   :members:
+
```

Comment:
I see `taskgen.py` added in another PR (https://github.com/LineaLabs/lineapy/pull/831). Does this mean the current PR is to be merged after that PR?

Reply:
yes